core/server/services/auth/session/index.js

```diff
@@ -0,0 +1,130 @@
+const {URL} = require('url'),
```

Comment:
>  core/server/services/auth/session/index.js

General note: we try to avoid having code in the `index.js` file.
The index.js file only offers the interface. It also can make use of magic getters to aim for more async/runtime loading.

Reply:
@allouis outstanding